src/OpenLoco/StationManager.cpp

```diff
@@ -53,8 +53,8 @@ namespace OpenLoco::StationManager
     {
         if (Game::hasFlags(1u << 0) && !isEditorMode())
         {
-            const auto id = StationId(scenarioTicks() & 0x3FF);
-            auto station = get(id);
+            const auto kId = StationId(scenarioTicks() & 0x3FF);
```

Comment:
Although this is constant it's not a constant known at compile time so i don't think it should have the k prefix

Reply:
done